docs/dev_guide/contents/newcomers.rst

```diff
@@ -127,9 +127,9 @@ Setting up a development environment
 
 The instructions in this following section are based upon three resources:
 
-* `Astropy Dev Workflow <https://docs.astropy.org/en/latest/development/workflow/development_workflow.html>`_
-* `Astropy Dev environment <https://docs.astropy.org/en/latest/development/workflow/get_devel_version.html#get-devel>`_
-* `Astropy Pull Request Example <https://docs.astropy.org/en/latest/development/workflow/git_edit_workflow_examples.html#astropy-fix-example>`_
+* `Astropy Dev Workflow <https://kbarbary-astropy.readthedocs.io/en/latest/development/workflow/development_workflow.html>`_
```

Comment:
```suggestion
```

Reply:
This page doesn't exist anymore.  We definitely shouldn't link to someone else's rendering of super-old Astropy docs.